xdocs/usermanual/functions.xml

```diff
@@ -1575,13 +1575,14 @@ returns:
     </description>
 <properties>
         <property name="Format" required="No">
-        The format to be passed to DateTimeFormatter.
+        The format to be passed to DateTimeFormatter (for input data parsing and output formating).
         See <a href="https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html">DateTimeFormatter</a>
-        If omitted, the function uses milliseconds since epoch format.
+        If omitted, the function returns milliseconds.
```

Comment:
While it returns milliseconds, those are anchored at start of the unix epoch. Hence I think that detail should not be dropped.

Reply:
Yes, You are right. I mean that when format is not given and date to shift is given (and must be Long) the code is reduced to result that don't  seem explicitly connect with epoch:

`Duration duration = Duration.parse("PT10S");`
`ZoneId systemDefaultZoneID = ZoneId.systemDefault();`
`ZonedDateTime  zonedDateTimeToShift = ZonedDateTime.ofInstant(Instant.ofEpochMilli(Long.parseLong("10000")), systemDefaultZoneID);`
`zonedDateTimeToShift = zonedDateTimeToShift.plus(duration);`
`String result = String.valueOf(zonedDateTimeToShift.toInstant().toEpochMilli());`
`System.out.println(result); //just 20000ms`


But, yes, when date to shift is not given  (is default: now) then connection with epoch is seen better:

`Duration duration = Duration.parse("PT10S");`
`ZoneId systemDefaultZoneID = ZoneId.systemDefault();`
`ZonedDateTime zonedDateTimeToShift = ZonedDateTime.now(systemDefaultZoneID);`
`zonedDateTimeToShift = zonedDateTimeToShift.plus(duration);`
`String result = String.valueOf(zonedDateTimeToShift.toInstant().toEpochMilli());`
`System.out.println(result); //1632157138893ms`